### PR TITLE
changed leaderboard text

### LINKED ISSUE #2989: On leaderboard, have "Want to improve your accuracy?"
On the leaderboard page, have something like "Want to improve your accuracy, read our labeling guide or view your recent incorrect labels on your dashboard" with links for each

### PR DESCRIPTION
**Pull request by lindseyehrlich**
Resolves #2989 

Changed the "Join an organization" text on the leaderboard page, adding links to /labelingGuide and /dashboard

##### Before/After screenshots (if applicable)
before

after

##### Testing instructions
1. 

##### Things to check before submitting the PR 

- [x] I've written a descriptive PR title. 
- [x] I've included before/after screenshots above.
- [x] I've asked for and included translations for any user facing text that was added or modified.

### CODE REVIEW

File: conf/messages.en-NZ [hunk 1]
```diff
@@ -42,6 +42,4 @@ results.legend.body = Gray neighbourhoods lack sufficient data to assess footpat
 dashboard.no.org = No Organisation
 dashboard.edit.info = If you don''t see your organisation, please email <a id="email-link" href="mailto:sidewalk@cs.uw.edu" target="_blank">sidewalk@cs.uw.edu</a> and include your organisation name and description.
 dashboard.orgs.unavailable = There are no available organisations!
-dashboard.leave.org = Leave Organisation
-
-leaderboard.encouragement.no.org = <a id="encouragement-link" href="/dashboard"><u>Join an organisation</u></a> and work together as a team to map and assess footpaths!
+dashboard.leave.org = Leave Organisation
```
Comment (misaugstad) [inline conf/messages.en-NZ:45]: Looks like you accidentally removed the blank line at the end of this file, which should be included, as per the style guide! Can you just add the blank line back?

**Review by misaugstad — CHANGES_REQUESTED** [review]
Looks good, just one issue with the style guide to fix!

I also added anchor links to the sections on the user dashboard so that we could link directly to the section on recent mistakes :)

**Comment by misaugstad** [comment]
Oh and @lindseyehrlich make sure to pull in the recent changes I made to your branch before you make more updates! So after you check out this branch, you just need to run `git pull` or `git pull origin 2989-leaderboard-text-change` to get the recent changes that I made!

**Review by misaugstad — APPROVED** [review]
Thanks @lindseyehrlich! First PR complete!

**Comment by lindseyehrlich** [comment]
Yay!!

**Pull request merged**